plugins/network/NetClass.cpp

```diff
@@ -2137,67 +2053,77 @@ void NetBrowser::ManualConnect()
 
 	if (PInfo.ItemsNumber)
 	{
-		size_t Size = PsInfo.PanelControl(this, FCTL_GETPANELITEM, PInfo.CurrentItem, {});
-		PluginPanelItem* PPI = (PluginPanelItem*)malloc(Size);
+		size_t Size = PsInfo.PanelControl(this, FCTL_GETPANELITEM, static_cast<intptr_t>(PInfo.CurrentItem), {});
+		auto* PPI = (PluginPanelItem*)malloc(Size);
 
 		if (PPI)
 		{
 			FarGetPluginPanelItem gpi = {sizeof(FarGetPluginPanelItem), Size, PPI};
-			PsInfo.PanelControl(this, FCTL_GETPANELITEM, PInfo.CurrentItem, &gpi);
+			PsInfo.PanelControl(this, FCTL_GETPANELITEM, static_cast<intptr_t>(PInfo.CurrentItem), &gpi);
 			ChangeToDirectory(PPI->FileName, OPM_NONE, true);
 			free(PPI);
 		}
 	}
 }
 
 
-void NetBrowser::GetRemoteName(NETRESOURCE* NetRes, wchar_t* RemoteName)
+string NetBrowser::GetRemoteName(NetResource* NetRes)
```

Comment:
If NetRes can't be null here, maybe accept by a const ref?

Reply:
ref, but not const